scrapy_spider_metadata/_params.py

```diff
@@ -22,11 +22,14 @@ def __init__(self, *args, **kwargs) -> None:
         super().__init__(*args, **kwargs)
 
     @classmethod
-    def get_param_schema(cls) -> Dict[Any, Any]:
+    def get_param_schema(cls, normalize=False) -> Dict[Any, Any]:
         """Return a :class:`dict` with the :ref:`parameter definition
         <define-params>` as `JSON Schema`_.
 
         .. _JSON Schema: https://json-schema.org/
+
+        If *normalize* is ``True``, the returned schema will be the same
+        regardless of whether you are using Pydantic 1.x or Pydantic 2.x.
```

Comment:
For the context, is it following pydantic 2.x output, pydantic 1.x output, or is it a different format?

Reply:
A different one. From Pydantic 1.x it takes:
- Not using `$defs`.
- Not using the enum class name as a default field `title`, and instead use the field name to generate such a default `title`.

I would mention the differences, but we cannot tell if future versions of Pydantic may remove some of them. Should we explicitly state that the output may not match either?

Reply:
> Should we explicitly state that the output may not match either?

I think so.